autonomy/fsm/scaffold/generators/components.py

```diff
@@ -70,11 +70,25 @@ def get_file_content(self) -> str:
             COPYRIGHT_HEADER,
             ROUNDS.HEADER.format(**self.template_kwargs),
             self._get_rounds_section(),
-            self.ABCI_APP_CLS.format(**self.template_kwargs),
+            self.ABCI_APP_CLS.format(
+                **self.template_kwargs,
+                db_pre_conditions=self._get_pre_condition_rounds_str(),
+                db_post_conditions=self._get_post_condition_rounds_str(),
+            ),
```

Comment:
Let's do this in the template instead:
https://github.com/valory-xyz/open-autonomy/blob/f4f5e704c2eee9e21deb6c7245282f8850554348/autonomy/fsm/scaffold/base.py#L119

Reply:
done